### PR TITLE
feat: make identifiers optional in company, update ChemicalElement

### PR DESCRIPTION
**Pull request by eamon0989**
# Description

Makes the `Identifiers` property optional in `company.json`.
Updates the tests for `company`.
Updates the PR template to wrap the description in comments.
Updates the properties in `ChemicalElement` to expect strings instead of numbers as explained in https://github.com/thematerials-network/EN10168-schemas/issues/44. 
Removes the `minimum` and `maximum` protections as they are JSON schema keywords and expect numbers and will not accept strings as can be seen here: https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.2.4.
Updates the tests for `ChemicalElement`.
Updates the README.

Fixes thematerials-network/EN10168-schemas#45

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] ⚠️ I ensured that the changes to the schema will be compatible with [schema-tools library](https://github.com/s1seven/schema-tools)

### CODE REVIEW

File: chemical-element/chemical-element.json [hunk 1]
```diff
@@ -13,28 +13,19 @@
         },
         "Actual": {
           "description": "The measured part of the element in percentage.",
-          "type": "number",
-          "minimum": 0,
-          "maximum": 100
+          "type": "string"
```
Comment (getlarge) [inline chemical-element/chemical-element.json:16]: even though it's not the most efficient way of doing it, you could replace minimum and maximum with a pattern ?
See [this example](https://regex101.com/r/d1Kdw5/1/)

Comment (eamon0989) [reply chemical-element/chemical-element.json:16]: Good idea!

Comment (eamon0989) [reply chemical-element/chemical-element.json]: I've added `^(?:[0-9]{1,2}(\\.\\d+)?|100)$`

**Comment by eamon0989** [comment]
> 

The most I've seen so far has been 4 decimal places if I remember correctly, but I'm not sure what the limit is. @stiebitzhofer should be able to provide a reasonable value.

**Comment by eamon0989** [comment]
> @stiebitzhofer maybe you would know what should be the precision required for Actual, Maximum and Minimum values in a Chemical Element ? @eamon0989 i guess right now there's no limitation on the decimals after the `.` (or `,`) with this pattern `^(?:[0-9]{1,2}(\\.\\d+)?|100)$` ?

Update: after talking with Hannes, we decided on limiting it to 4 decimal places. I've updated the pattern and tests.

File: README.md [hunk 2]
```diff
@@ -132,6 +132,8 @@ The Chemical Element definition is used by the EN10168 schema as a property in C
 | Minimum   | The minimum if defined by the product specification  |    No     |   Yes   |
 | Maximum   | The maxium as defined by the product specification   |    No     |   Yes   |
 
+WARNING: the minimum value of 0 and maximum value of 100 is no longer enforced by the JSON schema. Please ensure that these values are not exceeded, as they will not be flagged by the schema.
+
```
Comment (getlarge) [inline README.md:136]: this is no longer true with the regex used for validation ?
but maybe it's worthy to mention the 4 decimals limit

**Review by getlarge — COMMENTED** [review]
Looking good. That could have been nice to add some extra tests for the valid minimum/maximum values for chemical element

**Review by getlarge — APPROVED** [review]

**Review by stiebitzhofer — CHANGES_REQUESTED** [review]
Please add a valid company without `Identifiers` and without `Email`

**Review by stiebitzhofer — CHANGES_REQUESTED** [review]
See comments.

File: chemical-element/test/fixtures/invalid-certificate-7.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "Certificate": {
+    "ChemicalElement": {
+      "Maximum": "101",
+      "Symbol": "Ni",
+      "Minimum": "0",
```
[the remark this answers is not in the document]
Comment (eamon0989) [reply chemical-element/test/fixtures/invalid-certificate-7.json:6]: We don't currently have a test for that.

File: chemical-element/test/fixtures/valid-certificate-9.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "Certificate": {
+    "ChemicalElement": {
+      "Actual": "0.9999",
+      "Symbol": "Ni",
+      "Maximum": "1",
```
[the remark this answers is not in the document]
Comment (eamon0989) [reply chemical-element/test/fixtures/valid-certificate-9.json:6]: I've updated it so all have the same amount of decimal places.

File: README.md [hunk 5]
```diff
@@ -132,6 +132,8 @@ The Chemical Element definition is used by the EN10168 schema as a property in C
 | Minimum   | The minimum if defined by the product specification  |    No     |   Yes   |
```
[the remark this answers is not in the document]
Comment (eamon0989) [reply README.md:132]: In English units, it would be after the dot rather than the comma. Since the documentation is in English, is it ok if I put "after the dot"?

[the remark this answers is not in the document]
Comment (eamon0989) [reply README.md:132]: To be even more precise, I would say "after the decimal separator".

File: chemical-element/test/fixtures/invalid-certificate-7.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (getlarge) [reply chemical-element/test/fixtures/invalid-certificate-7.json:6]: When you say length you meant number of digits after the comma ? 
I doubt it's possible to check the value of other properties only with standards JSON schema validators.

**Review by stiebitzhofer — APPROVED** [review]
